auth/src/api/services/user-service.ts

```diff
@@ -32,10 +32,7 @@ export class UserService implements UserHandler {
         if (!user) {
             return { isValid: false, message: 'Username or password is incorrect' };
         }
-        if (
-            (!user.password && user.default_password.slice(32) !== this.hashingHandler.hash(password)) ||
-            (user.password && user.password.slice(32) !== this.hashingHandler.hash(password))
-        ) {
+        if (!this.isPasswordCorrect(password, user.password || user.default_password)) {
```

Comment:
is `default_password` the user field from the db?

do you make chekcs, that `password` is well formatted: Has it the right length?

Reply:
@GabrielInTheWorld 

> is default_password the user field from the db?

Reply:
> is `default_password` the user field from the db?

Yes, it is.

Reply:
You should not be interested in this field. It is neither hashed nor filled in a way the auth service controls. See #36 . Can be done in another PR.